### ISSUE TITLE
[Global] Does not support landscape orientation

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#11]
## Issue summary

The app doesn't seem to support landscape orientation on Android devices. On iOS, it's partially supported though some text nodes are not visible. Supporting landscape orientation is important for people who prefer one orientation to another. Some may not have the physical capability of making an adjustment to their device. Supporting only portrait mode could lead to a frustrating user experience.

<img width="378" alt="Screen Shot 2020-05-22 at 11 56 55 AM" src="https://user-images.githubusercontent.com/1392632/82703639-50776d00-9c42-11ea-8ebe-7d9811b8f133.png">
![Screen Shot 2020-05-22 at 3 38 15 PM](https://user-images.githubusercontent.com/1392632/82703642-51100380-9c42-11ea-94e1-c4498c4ace5f.png)

## Steps to reproduce

1. Install the CovidShield `apk` on an Android device
2. Load the app
3. Adjust to landscape orientation

### Expected

- Content to adjust to landscape mode, allowing full use of the app.

### Actual

- App does not switch to landscape mode.

## Recommendation

Allow any orientation. Ensure app content, layout, and controls are accessible when switching device orientation.

## Specifications

- **Component:** CovidShield [Global]
- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.3.4 Orientation
- **Severity:** High
- **Effort:** Unknown

**Comment by adam-poteralowicz** [comment]
Could I take on that issue?

**Comment by AdamWhitcroft** [comment]
Recommendations for better landscape support.

**Onboarding screens, steps 1 and 2**

![1](https://user-images.githubusercontent.com/861526/83036053-33f38000-a008-11ea-81f6-cd8dacc652b4.png)

![2](https://user-images.githubusercontent.com/861526/83036056-348c1680-a008-11ea-8eb6-1b36541e7ae1.png)

**Onboarding, learn how**

In order to maximise screen real-estate for written content, I'm proposing we drop the animations from these slides. Including them would mean either putting it above or below the text. Above would push the content out of view, and below means you might not even know they are there.

![3](https://user-images.githubusercontent.com/861526/83036211-54bbd580-a008-11ea-8a32-3bbacba6ea59.png)

![4](https://user-images.githubusercontent.com/861526/83036267-656c4b80-a008-11ea-87c0-1655afc68c30.png)

![5](https://user-images.githubusercontent.com/861526/83036326-74eb9480-a008-11ea-85d4-0c8683a06040.png)

**Thank you screen**

Sheet button 400pt wide.

![6](https://user-images.githubusercontent.com/861526/83036623-d7449500-a008-11ea-858e-c9f185a8c80f.png)

**Home, No exposure detected**

![Frame 105](https://user-images.githubusercontent.com/861526/83040056-d7df2a80-a00c-11ea-8b8c-c2fcb994fc14.png)

**Home, Sheet shown**

![Frame 105](https://user-images.githubusercontent.com/861526/83040230-0957f600-a00d-11ea-9fb6-b6652d1d14b3.png)

**Home, Sheet callout**

Use a 24pt "safe space" at the top of the content area to keep header and content visually distinct. 

![Frame 105](https://user-images.githubusercontent.com/861526/83038702-43c09380-a00b-11ea-9f66-a807e5930efa.png)

**Home, Potentially exposed / thank you**

Keeping the animations these two scenarios comes at a significant cost. The main Call To Action for each stage may well be pushed completely out of view. If we feel strongly the animations are important in this orientation, we should render an alternative (smaller) version of them).

Button 400pt wide.

![Frame 105](https://user-images.githubusercontent.com/861526/83041311-57212e00-a00e-11ea-9d30-1f1e2d661503.png)

And without animation (my preference).

![Frame 105](https://user-images.githubusercontent.com/861526/83042111-445b2900-a00f-11ea-8a43-bab093f17191.png)

**Home, High alert**

![Frame 105](https://user-images.githubusercontent.com/861526/83043809-50e08100-a011-11ea-8ab2-377bef2d5dd4.png)

**Code entry**

![Frame 105](https://user-images.githubusercontent.com/861526/83044164-d106e680-a011-11ea-8042-8a48222e6fd1.png)

**Code entry, Consent**

![Frame 105](https://user-images.githubusercontent.com/861526/83044374-15928200-a012-11ea-9274-f26600ace7f7.png)

**Comment by henrytao-me** [comment]
Excellent works @AdamWhitcroft 👍

**Comment by kpeatt** [comment]
@AdamWhitcroft Can we possibly keep the animations in the tutorial by going to a columnar layout?

**Comment by AdamWhitcroft** [comment]
@kpeatt it might work if we scale down the animations (and maybe re-render them as we have clipping of the echoes on the left and right sides)

![Screen Recording 2020-05-27 at 2 35 56 PM](https://user-images.githubusercontent.com/861526/83059240-9b6cf800-a027-11ea-9050-b59fbd818dcc.gif)

![Screen Recording 2020-05-27 at 2 33 10 PM](https://user-images.githubusercontent.com/861526/83059253-9f007f00-a027-11ea-8cd5-a033c218edee.gif)

**Comment by kpeatt** [comment]
The above looks good to me.
